.github/workflows/linter.yml

```diff
@@ -0,0 +1,20 @@
+name: "Linter"
+
+on: [pull_request]
+jobs:
+  lint:
+    name: Linter
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 2
+
+    - run: git checkout HEAD^2
```

Comment:
same comments as above

Reply:
Not sure, copied it from somewhere. Replacing with what we have in release.yml, without this twos.